src/components/Sidebar/index.tsx

```diff
@@ -19,6 +20,7 @@ const Sidebar: React.FC = () => {
       py={4}
       px={5}
       w="100%"
+      h="100%"
```

Comment:
minH

Reply:
It is not working due, anyhow any particular reason for minH ? as I think we want h to be 100% always